hubstorage/jobq.py

```diff
@@ -31,6 +31,13 @@ def push(self, spider, **jobparams):
                 raise DuplicateJobError()
             raise
 
+    def jobsummary(self, jobkeys, jobmeta):
+        """Fetch selected job metadata fields for selected jobs."""
+        if not isinstance(jobkeys, (list, tuple)):
+            raise TypeError("jobkeys must be a list or a tuple")
+        return list(self.apiget(('jobsummary',),
```

Comment:
Other methods return a generator, shouldn't we follow that convention and return `self.apiget` result without a `list`?